server/task-distributed/pom.xml

```diff
@@ -28,8 +28,8 @@
         <version>3.4.0-SNAPSHOT</version>
     </parent>
 
-    <artifactId>james-server-task-cassandra</artifactId>
-    <name>Apache James :: Server :: Task :: Cassandra</name>
+    <artifactId>james-server-task-distributed</artifactId>
+    <name>Apache James :: Server :: Task :: Distributed</name>
```

Comment:
No description? Can we explain that this distributed manager leverages Cassandra & RabbitMQ ?